bazel_tools/nixpkgs-java-toolchain.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/nixpkgs/nixpkgs.bzl b/nixpkgs/nixpkgs.bzl
```

Comment:
I’ll work on upstreaming a slightly cleaned up version of this.

Reply:
for reference this is the upstream PR https://github.com/tweag/rules_nixpkgs/pull/185